src/sync/installation.ts

```diff
@@ -225,6 +432,12 @@ export const processInstallation =
 				jiraHost
 			});
 
+			if (await booleanFlag(BooleanFlags.SPLIT_PROCESS_INSTALLATION_FUNCTION, false, jiraHost)) {
+				return await doProcessInstallation(app, queues, job, installationId, jiraHost, logger);
```

Comment:
[Return await is redundant](https://eslint.org/docs/rules/no-return-await)

- [ ] update return: call await func(), followed by return;